src/sidebar/store/modules/annotations.js

```diff
@@ -415,10 +417,9 @@ function updateFlagStatus(id, isFlagged) {
 
 /**
  * Count the number of annotations (as opposed to notes or orphans)
- *
- * @type {(state: any) => number}
  */
 const annotationCount = createSelector(
+  /** @param {State} state */
```

Comment:
Specifying the type here allows everything else, including the return type when the selector is called, to be inferred. In some instances it might be useful to explicitly write out what the return type is, but I haven't figured out the best way to do that yet (`@returns` doesn't work because `createSelector` calls are not functions).

Reply:
I...see...sure...a bit ungainly, but yep.